src/lib/I18n.js

```diff
@@ -0,0 +1,89 @@
+/**
```

Comment:
Yes I started like that but I came back because `cozy-ui` needs a dependency to `date-fns` which is just 80kb (production build) of useless code for the cozy-bar here. So this is all the code from cozy-ui without the `date-fns` and `format` parts.